roles/edpm_build_images/README.md

```diff
@@ -9,10 +9,15 @@ the variables "cifmw_edpm_build_images_ironic_python_agent_package" and
 ## Privilege escalation
 None
 
+## Vars
+These variables should not be updated.
+* `cifmw_edpm_build_images_host_packages`: List of packages required to build the images.
+
+
 ## Parameters
 * `cifmw_edpm_build_images_basedir`: Base directory. Defaults to `cifmw_basedir` which  defaults to `~/ci-framework`.
 * `cifmw_edpm_build_images_via_rpm`: Whether to install `edpm-image-builder` repo using rpm or not.
-* `cifmw_build_host_packages`: List of packages required to build the images.
```

Comment:
cifmw_edpm_build_images_host_packages - maybe it should be that var rather than removing it? or cifmw_edpm_build_images_host_packages_default ... and

cifmw_edpm_build_images_host_packages = cifmw_edpm_build_images_host_packages_default + cifmw_edpm_build_images_host_extra_packages